File: grouping-client/src/app/models/groups.ts
export class Groups {

  constructor(
    private _id: number,
    private _name: string,
    private _groupAdmin: number,
    private _tocken: string | null
  ) {}

  public get id(): number {
    return this._id;
  }
  public set id(value: number) {
    this._id = value;
  }
  public get name(): string {
    return this._name;
  }
  public set name(value: string) {
    this._name = value;
  }
  public get groupAdmin(): number {
    return this._groupAdmin;
  }
  public set groupAdmin(value: number) {
    this._groupAdmin = value;
  }

  public get tocken(): string | null {
    return this._tocken;
  }
  public set tocken(value: string | null) {
    this._tocken = value;
  }
}

File: groups-service/src/main/java/com/ironhack/groupsservice/model/Groups.java
package com.ironhack.groupsservice.model;

import javax.persistence.*;
import java.util.List;

@Entity
public class Groups {

    /**
     * Properties
     */
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Long id;
    private Long groupAdmin;

    @OneToMany(mappedBy = "groups")
    private List<GroupsMembers> groupsMembersList;

    /**
     * Default constructor
     */
    public Groups() {
    }

    /**
     * Constructor: groupAdmin
     */
    public Groups(Long groupAdmin) {
        this.groupAdmin = groupAdmin;
    }

    /**
     * Getters & Setters
     */
    public Long getId() {
        return id;
    }

    public void setId(Long id) {
        this.id = id;
    }

    public Long getGroupAdmin() {
        return groupAdmin;
    }

    public void setGroupAdmin(Long groupAdmin) {
        this.groupAdmin = groupAdmin;
    }

    public List<GroupsMembers> getGroupsMembersList() {
        return groupsMembersList;
    }

    public void setGroupsMembersList(List<GroupsMembers> groupsMembersList) {
        this.groupsMembersList = groupsMembersList;
    }
}
